app/src/app/[lng]/[inventory]/data/[step]/page.tsx

```diff
@@ -624,21 +624,24 @@ export default function AddDataSteps({
   return (
     <>
       <Box id="top" />
-      <MotionBox
+      <Box
         bg="background.backgroundLight"
         borderColor="border.neutral"
         borderBottomWidth={scrollPosition > 0 ? "1px" : ""}
-        className={`fixed z-10 top-0 w-full pt-[130px] h-[400px]`}
-        mt={scrollPosition > 0 ? "-200px" : ""}
-        animate={{
-          y: scrollPosition > 0 ? 0 : -50,
-        }}
-        transition={{ duration: 0.2 }}
+        className={`fixed z-10 top-0 w-full ${scrollPosition > 0 ? "pt-[0px] h-[200px]" : "pt-[120px] h-[400px]"} transition-all duration-50 ease-linear`}
       >
-        <div className=" w-[1090px] max-w-full mx-auto px-4">
-          <Box w="full" display="flex" alignItems="center" gap="16px" mb="35px">
+        <div className=" w-[1090px] mx-auto px-4  ">
+          <Box
+            w="full"
+            display="flex"
+            alignItems="center"
+            gap="16px"
+            mb="28px"
+            className={` ${scrollPosition > 0 ? "hidden" : "flex"} transition-all duration-50 ease-linear`}
```

Comment:
You use scrollPosition > 0 so much, it might be worth defining a variable for it above  
E.g. `const isExpanded = scrollPosition > scrollResizeHeaderThreshold;`